### DISCUSSION TITLE
Help with evocating with quotes

### DISCUSSION THREAD

**Discussion by Raizanad** [PowerShell/PowerShell#20835] [Q&A]
Someone knows what I may have done wrong, or what program or setting might have changed powershell behavior so that I can't run commands like "C:\Program Files\Git\cmd\git.exe" with quotation to choose an exe path? I'm unable to run scripts as a result. Git and Python run fine typing git or python or py alone, and both methods work in cmd but not on powershell, powershell preview or windows powershell.
![Screenshot 2023-12-03 025255](https://github.com/PowerShell/PowerShell/assets/131222736/07323bbe-640c-4130-a4c8-835041380a3f)

I know python -C likely does nothing but the image above is for describing what I'm experiencing as google search are yielding no results.
![Screenshot 2023-12-03 030003](https://github.com/PowerShell/PowerShell/assets/131222736/d00d0e55-2031-4cb1-9ef4-eeb12c031429)
![Screenshot 2023-12-03 025924](https://github.com/PowerShell/PowerShell/assets/131222736/41093619-27c1-4bae-9dec-df7974b3d214)
![Screenshot 2023-12-03 025851](https://github.com/PowerShell/PowerShell/assets/131222736/e3e3cd43-66a6-415b-8d8b-60410089c736)

It was working fine around 1 or 2 weeks ago, I'm genuinely stumped.

**Comment by MartinGC94** [comment]
Look closer at your screenshots. In the screenshots where it work you have `&` in front of the quoted string.  
This is because PowerShell needs the `&` (or `.`) character to let it know that the string should be treated as a command, and not just a literal string.

**Reply by Raizanad** [reply]
I think it may be my memory that is wrong then. I seemed to recall that the & wasn't needed as far as 2 weeks ago, but the literature reads differently. I can't figure out why all my scripts, and stable diffusion aren't working anymore,
